includes/class-newspack-ads-placements.php

```diff
@@ -457,6 +457,45 @@ public static function disable_placement( $placement_key ) {
 		);
 	}
 
+	/**
+	 * Whether the placement can display an ad unit.
+	 *
+	 * @param string $placement_key Placement key.
+	 *
+	 * @return bool Whether the placement can display an ad unit.
+	 */
+	public static function can_display_ad_unit( $placement_key ) {
+		$can_display = true;
```

Comment:
I agree that this nesting is not ideal. The main reason for it was that I wanted to preserve a single exit point for `apply_filters().`. I don't think the reverse of `$can_display` helps too much in this case.

f80744f19e0261f1ef600d0b6992ae29fea3183e isolates the logic in a private method so it can exit early to improve readability and the public method applies the filter.

Reply:
Got it—I do think this is slightly more readable, and the added comments help. Thanks!